rfc9002.md

```diff
@@ -345,11 +345,11 @@ path's actual RTT increases however, the min_rtt will not adapt to it, allowing
 future RTT samples that are smaller than the new RTT to be included in
 smoothed_rtt.
 
-Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent
-congestion is established. This is to allow a connection to reset its estimate
-of min_rtt and smoothed_rtt ({{smoothed-rtt}}) after a disruptive network event,
-and because it is possible that an increase in path delay resulted in persistent
-congestion being incorrectly declared.
+Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent
```

Comment:
```suggestion
Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent
```

Reply:
These two lines look identical, but I expect they're not and I copied over a non-breaking space or something?